<!DOCTYPE HTML>
<!--
	Phantom by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="inner">

							<!-- Logo -->
								<a href="index.html" class="logo">
									<span class="symbol"><img src="images/logo.svg" alt="" /></span><span class="title">Owens</span>
								</a>

							<!-- Nav -->
								<nav>
									<ul>
										<li><a href="#menu">Menu</a></li>
									</ul>
								</nav>

						</div>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<h2>Menu</h2>
						<ul>
                            <li><a href="index.html">Home</a></li>
                            <li><a href="research.html">Research</a></li>
                            <li><a href="people.html">People</a></li>
                            <li><a href="publications.html">Publications</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">
						<div class="inner">
							<h1>Research</h1>
							<span class="image main"><img src="images/Dynamic-edge.svg" alt="" /></span>
                            <p>
                            <h3>Inversions</h3>
                            <span class="image left"><img src="images/inversion_icon.png" alt="" /></span>
                            Local adaptation often requires the effect of multiple separate loci. This raises the question, how do locally adaptive loci stay together in the face of maladaptive gene flow and recombination. One solution are inversions which can link together loci across large genomic regions.
                            <br>
                            In sunflowers, we have found dozens of large (1-100MBp) inversions segregating within populations. These regions are important in local adaptation and we've shown associations between inversion haplotypes and flowering time as well as seed size. We're working to understand the evolutionary history of these inversions and develop better methods to detect them in genomic sequence data.
                            </p>
                            
                            <hr>
                            <p>
                           <h3 align="left">Hybridization</h3><p><span class="image left"><img src="images/hybridization_icon.png" alt="" /></span>
                           The exchange of alleles between species through hybridization is an important aspect of evolution. Hybridization has enhanced diversification in rapidly speciating radiations and can spread adaptive alleles between species. We've used genomic data to detect introgression (foreign alleles) in multiple sunflower species as well as the role it's played in domestic sunflower improvement.
                           <br>
                           We use genomic data to quantify hybridization across two rapidly radiating sets of species, sunflowers and rockfish. We're interested in where and why hybridization occurs and what alleles do or do not transfer between species.
                            
                            </p>
                            <hr>
                            <p>
                            <h3>Crop-wild diversity</h3>
                            <span class="image left"><img src="images/cropwild_icon.png" alt="" /></span>
                            During domestication, genetic diversity is lost. This can cause issues when crop species are forced to adapt to new diseases or climate pressures. One solution is to introduce diversity from crop-wild relatives, this includes the wild progenitor to the domestic species and close relatives. Understanding the population genetics of these crop-wild relatives allows plant breeders to make smart choices when bringing wild species into the breeding pool.
                            <br>
                            We focus on sunflowers and blueberries, two species domesticated in North America. We use genomics to understand the relationships between wild species and to identify alleles associated with climate, potential candidates for future breeding.
                            <hr>
                            </p>
                            <p>

                           <h3 align="left">Visual adaptation</h3> <span class="image left"><img src="images/vision_icon.png" alt="" /></span>
                            Opsin proteins are the molecular mechanism for light perception and vision. Evolution has acted on vision by tuning which wavelengths each opsin protein absorbs and by changing opsin gene expression. Studying opsin evolution provides a unique opportunity to link genotypic changes to functional consequences.
                            <br>
                            We study opsin gene repertoire and expression in fish. Our previous work has shown the in the four-eyed fish, <i>Anableps anableps</i>, different opsin genes are expressed in parts of the retina. We've also found that during freshwater colonization stickleback change their opsin expression to match the available light. We have ongoing work to examine how rockfish have adapted their opsin genes to divergent habitat and body colouration patterns.
                            </p>
                        </div>
					</div>

				<!-- Footer -->
					                    <footer id="footer">
                        <div class="inner">
                            
                            <section>
                                <h2>Follow</h2>
                                <ul class="icons">
                                    <li><a href="https://twitter.com/Greg_Owens" class="icon brands style2 fa-twitter"><span class="label">Twitter</span></a></li>
                                    <li><a href="https://github.com/owensgl" class="icon brands style2 fa-github"><span class="label">GitHub</span></a></li>
                                    <li><a href="mailto: gregory.owens@berkeley.edu" class="icon solid style2 fa-envelope"><span class="label">Email</span></a></li>
                                </ul>
                            </section>
                            <ul class="copyright">
                                <li>&copy; Gregory L Owens. All rights reserved</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
                            </ul>
                        </div>
                    </footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
